### PR TITLE
ALFREDAPI-556 [Update] package name-change from 'eu.xenit.apix' to 'e…

### PR DESCRIPTION
**Pull request by codingBenVdS**
ALFREDAPI-556 [Update] package name-change from 'eu.xenit.apix' to 'eu.xenit.alfred.api'

Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-556

- [x] Is [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md) extended?
- [ ] Does this PR avoid breaking the API? 
    Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
- [x] Does the PR comply to REST HTTP result codes policy outlined in the [user guide](https://docs.xenit.eu/alfred-api/user/rest-api/index.html#rest-http-result-codes)?
- [x] Is error handling done through a method annotated with `@ExceptionHandler` in the webscript classes?
- [x] Does the PR follow our [coding styleguide and other active procedures](https://xenitsupport.jira.com/wiki/spaces/XEN/pages/624558081/XeniT+Enhancement+Proposals+XEP)?
- [x] Is usage of `this.` prefix avoided?

See [README.md](./README.md) for full explanation.

### CODE REVIEW

File: apix-impl/src/main/java/eu/xenit/alfred/api/Version.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package eu.xenit.alfred.api;
```
Comment (todorinskiz) [inline apix-impl/src/main/java/eu/xenit/alfred/api/Version.java:1]: I'm not really happy with this file. Can't we use reflection?

```
YourClass.class.getPackage().getSpecificationVersion();
```

**Review by todorinskiz — COMMENTED** [review]
minor remark

**Comment by todorinskiz** [comment]
How far are we willing to go here?
There is still:

- test model with `apix`
- URL path starts with `/apix/...`
- gradle projects are also `apix-...`

**Comment by WimCrols** [comment]
> How far are we willing to go here? There is still:
> 
> * test model with `apix`
> * URL path starts with `/apix/...`
> * gradle projects are also `apix-...`

True... you to stop somewhere. I'm open to adjusting internal names, but URL I would not touch. Maybe out of scope for this ticket.

We also may need to doublecheck artifact names...

Also

**Review by WimCrols — CHANGES_REQUESTED** [review]

File: apix-impl/src/main/java/eu/xenit/alfred/api/Version.java
[the remark this answers is not in the document]
Comment (WimCrols) [reply apix-impl/src/main/java/eu/xenit/alfred/api/Version.java:1]: How did you manage to get this committed? It's ignored in https://github.com/xenit-eu/alfred-api/blob/master/.gitignore#L2

[the remark this answers is not in the document]
Comment (WimCrols) [reply apix-impl/src/main/java/eu/xenit/alfred/api/Version.java:1]: Ah, I see... the path has changed and needs to be adjusted.

[the remark this answers is not in the document]
Comment (codingBenVdS) [reply apix-impl/src/main/java/eu/xenit/alfred/api/Version.java:1]: i looked over this. I'll adjust whats necessary

**Comment by RVanhuysseXenit** [comment]
> How far are we willing to go here? There is still:
> 
>     * test model with `apix`
> 
>     * URL path starts with `/apix/...`
> 
>     * gradle projects are also `apix-...`

For this particular issue i was just hoping for package names. There's 2 reasons:
* code generation becomes easier. without this change there's still some manual/bulk fiddling to align the interface object with generated obejcts
* you avoid confusion for devs that use the interface as a lib for their client, pdl had issues with this. Having to explain the old situation created some friction. With a less easygiong client this would have been a big issue.

File: apix-interface/src/main/java/eu/xenit/apix/dictionary/properties/IPropertyService.java
[the remark this answers is not in the document]
Comment (codingBenVdS) [reply apix-interface/src/main/java/eu/xenit/apix/dictionary/properties/IPropertyService.java:1]: that's correct.
image

File: apix-impl/src/main/java/eu/xenit/alfred/api/Version.java
[the remark this answers is not in the document]
Comment (codingBenVdS) [reply apix-impl/src/main/java/eu/xenit/alfred/api/Version.java:1]: Removed the file and updated .gitignore

**Comment by codingBenVdS** [comment]
Changed all the apix references from the (Folders || Classnames || Variables || ..)

**Review by WimCrols — APPROVED** [review]

**Review by todorinskiz — APPROVED** [review]

**Pull request merged**